### PR TITLE
Ha pair SBD setup

### PR DESCRIPTION
**Pull request by pabowers**
This PR sets up the iSCSI and SBD devices that are needed for the pacemaker.  The HANA install will happen in another PR.  Pacemaker setup follows the procedure described [here](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker).

### CODE REVIEW

File: experiment/ansible/roles/set-up-sbd-device/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+cluster_name: cl1
+iscsi_object: iqn.2006-04
+iscsi_ip: 10.0.0.17
+iscsi_port: 3260
```
[the remark this answers is not in the document]
Comment (pabowers) [reply experiment/ansible/roles/set-up-sbd-device/defaults/main.yml]: sorry, went back and did this in all places

File: experiment/ansible/roles/iscsi-setup/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    state: absent
+
+- name: Remove unnecessary python-rtslib
+  package:
+    name: python-rtslib
+    state: absent
+
+- name: Remove unnecessary old python-configshell
+  package:
+    name: python-configshell
```
Comment (persiaAziz) [inline experiment/ansible/roles/iscsi-setup/tasks/main.yml]: How does it know python-configshell is old? or it just removes any version there is and installs the new one?

File: experiment/ansible/roles/iscsi-setup/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    msg: "{{ lun_to_iscsi_object }}"
+  when: lun_to_iscsi_object
+
+- name: Add acl 0
+  command: targetcli iscsi/{{ iscsi_object }}.{{ cluster_name }}.local:{{ cluster_name }}/tpg1/acls/ create {{ iscsi_object }}.prod-{{ cluster_name }}-0.local:prod-{{ cluster_name }}-0
+  when: lun_to_iscsi_object.changed
+  register: add_acl0
+
+- debug:
+    msg: "{{ add_acl0 }}"
```
Comment (persiaAziz) [inline experiment/ansible/roles/iscsi-setup/tasks/main.yml]: is the msg for print debugging?

Comment (pabowers) [reply experiment/ansible/roles/iscsi-setup/tasks/main.yml]: yes.  I can remove those to clean up the code.

File: experiment/ansible/roles/set-up-sbd-device/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,103 @@
+- name: Enable iscsid
+  service: name=iscsid enabled=yes
+
+- name: Enable iscsi
+  service: name=iscsi enabled=yes
+
+- name: Enable sbd
```
Comment (persiaAziz) [inline experiment/ansible/roles/set-up-sbd-device/tasks/main.yml]: When does the sbd service actually get installed?

File: experiment/ansible/roles/set-up-sbd-device/tasks/main.yml [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+- name: Adapt the SBD config to have SDB startmode set to always
+  lineinfile:
+    dest: '/etc/sysconfig/sbd'
+    regexp: 'SBD_STARTMODE='
+    line: "SBD_STARTMODE=\"always\""
+    state: 'present'
+  register: startmode_set
+  
+- name: Create softdog config
```
Comment (persiaAziz) [inline experiment/ansible/roles/set-up-sbd-device/tasks/main.yml]: how does the softdog module get installed?

File: experiment/ansible/roles/set-up-sbd-device/tasks/main.yml [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+  command: echo softdog | sudo tee /etc/modules-load.d/softdog.conf
+
+- name: Load the softdog module
```
Comment (persiaAziz) [inline experiment/ansible/roles/set-up-sbd-device/tasks/main.yml]: how does the softdog module get installed?

Comment (persiaAziz) [reply experiment/ansible/roles/set-up-sbd-device/tasks/main.yml]: same question for other commands, services used here

File: experiment/ansible/roles/set-up-sbd-device/tasks/main.yml (hunk 4 above)
Comment (pabowers) [reply experiment/ansible/roles/set-up-sbd-device/tasks/main.yml]: It comes installed on a SLES vm

File: experiment/ansible/roles/set-up-sbd-device/tasks/main.yml (hunk 6 above)
Comment (pabowers) [reply experiment/ansible/roles/set-up-sbd-device/tasks/main.yml]: iscsi and iscsid services are on the machine already, but need to be enabled.  Similarly, softdog is also already on the machine.

**Pull request merged**